README.md

```diff
@@ -60,7 +112,7 @@ This reference architecture implementation uses Terraform. You will need to do t
    terraform apply
    ```
 
```

Comment:
Add a heading  `## Terraform docs` like in the [AWS README](https://github.com/humanitec-architecture/reference-architecture-aws/blob/38d045a397b39672fd3e6ef2e6a9b965ec201d62/README.md?plain=1#L132)

Reply:
Added, please check